cmd/limactl/gendoc.go

```diff
@@ -2,30 +2,33 @@ package main
 
 import (
 	"bytes"
+	"fmt"
 	"io/fs"
 	"os"
 	"path/filepath"
+	"strings"
 
 	"github.com/cpuguy83/go-md2man/v2/md2man"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"github.com/spf13/cobra/doc"
 )
 
-func newGenManCommand() *cobra.Command {
+func newGenDocCommand() *cobra.Command {
 	genmanCommand := &cobra.Command{
-		Use:    "generate-man DIR",
-		Short:  "Generate manual pages",
+		Use:    "generate-doc DIR",
+		Short:  "Generate cli-reference pages",
```

Comment:
It might not be necessary to rename the command.
Docsy manuals are still manuals, but in a different format.

Reply:
The command is internal anyway, so it is not important.

Those were just the command names that I used, for them...

Reply:
Yes exactly. I didn't want to have that confusion with man. 
So just renamed it to more generic one so its clear.